File: Beecrowd/Beginner/2203 - ~~/2235 - Walking in Time/C++/2235_WalkinginTime.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int credit1, credit2, credit3;
    cin >> credit1 >> credit2 >> credit3;

    if (credit1 == credit2 || credit1 == credit3 || credit2 == credit3 || credit1 + credit2 == credit3 || credit1 + credit3 == credit2 || credit2 + credit3 == credit1) cout << "S" << endl;
    else cout << "N" << endl;

    return 0;
}

File: Beecrowd/Beginner/2203 - ~~/2235 - Walking in Time/Python/2235_WalkinginTime.py
credit1, credit2, credit3 = map(int, input().split())

if (credit1 == credit2 or credit1 == credit3 or
    credit2 == credit3 or credit1 + credit2 == credit3 or
    credit1 + credit3 == credit2 or credit2 + credit3 == credit1):
    print("S")
else:
    print("N")
